utils/spacewalk-manage-channel-lifecycle

```diff
@@ -320,14 +320,24 @@ def build_channel_labels(source):
             sys.exit(1)
     elif options.promote:
         # get the phase label from the channel label
-        current_phase = get_current_phase(source)
-        if current_phase:
-            next_phase = get_next_phase(current_phase)
-
-            # replace the name of the phase in the destination label
-            destination = re.sub('^%s' % current_phase, next_phase, source)
+        match = re.search('^(\w+)-', source)
+        if match:
```

Comment:
This is wrong. It have to respect --delimiter option.
I'd also prefer to fix get_current_phase() instead of a lot of new code here.